yt_dlp/downloader/http.py

```diff
@@ -213,6 +213,11 @@ def close_stream():
         def download():
             data_len = ctx.data.info().get('Content-length', None)
 
+            if ctx.data.info().get('Content-encoding', None):
```

Comment:
`None` is default?
```suggestion
            if ctx.data.info().get('Content-encoding'):
```

Reply:
Yes, I'm just trying to keep the same coding style as data_len is also adding this `, None`. I'll try to remove both then.